include/deal.II/base/discrete_time.h

```diff
@@ -103,11 +114,12 @@ DEAL_II_NAMESPACE_OPEN
  *   size $t_{n+1} - t_n$ and *previous* time step size $t_n - t_{n-1}$. In
  *   this stage, it is a perfect occasion to generate text output using print
  *   commands within the user's code. Additionally, post-processed outputs can
- *   be prepared here which can be viewed later by visualization programs such
- *   as `Tecplot`, `Paraview`, and `VisIt`. Additionally, during the snapshot
- *   stage, the code can assess the quality of the previous step and decide
- *   whether it wants to increase or decrease the time step size. The step
- *   size for the next time step can be modified here.
+ *   be prepared here, which can then later be viewed by visualization programs
+ *   such as `Tecplot`, `Paraview`, and `VisIt`. Additionally, during the
+ *   snapshot stage, the code can assess the quality of the previous step and
+ *   decide whether it wants to increase or decrease the time step size. The
+ *   step size for the next time step can be modified here, by calling
+ *   DiscreteTime::set_desired_next_step_size().
```

Comment:
```suggestion
 *   set_desired_next_step_size().
```
Name of the class is not needed here and we don't have the class name prefix when referring to other methods. We can remove the prefix but It is totally fine if we keep it.